template/$PROJECT_NAME$/lib/$PROJECT_NAME$/account/user.ex

```diff
@@ -23,7 +23,7 @@ defmodule <%= @project_name_camel_case %>.Account.User do
   end
 
   @spec hash_password(%Ecto.Changeset{}) :: %Ecto.Changeset{}
-  defp hash_password(changeset = %Ecto.Changeset{valid?: true, changes: %{password: pass}}) do
+  defp hash_password(%Ecto.Changeset{valid?: true, changes: %{password: pass}} = changeset) do
```

Comment:
I got it from @daveshah. Pretty sure it came from Credo. He changed it in Agoneum when he added Credo to the project